editor/import/resource_importer_scene.cpp

```diff
@@ -313,7 +313,7 @@ static bool _teststr(const String &p_what, const String &p_str) {
 	String what = p_what;
 
 	//remove trailing spaces and numbers, some apps like blender add ".number" to duplicates so also compensate for this
-	while (what.length() && (is_digit(what[what.length() - 1]) || what[what.length() - 1] <= 32 || what[what.length() - 1] == '.')) {
+	while (what.length() && (is_digit(what[what.length() - 1]) || what[what.length() - 1] <= 32 || what[what.length() - 1] == '_')) {
```

Comment:
GH won't let you comment on the right line but: The comment should be amended to clarify, it's talking about `.number` and we're handling `_number`.

I'm also a bit surprised that the fix is this, reduz's PR changed it to replace `.` by `_` instead of an empty string. But here it sounds like the previous logic still handled `.` so it happened _before_ the replacement, and now it seems to happen after?

Reply:
I think he did it right :) I can just replace the old `hint.num3er` with `hint_num3er`.

Reply:
?
// Remove trailing spaces and numbers, some apps like blender add ".number" 
// (dot is replaced by _ as invalid character) to duplicates so also compensate for this.

Reply:
Sounds good.